app/decorators/controllers/solidus_braintree/carts_controller_decorator.rb

```diff
@@ -1,11 +1,11 @@
 # frozen_string_literal: true
 
 module SolidusBraintree
-  module OrdersControllerDecorator
+  module CartsControllerDecorator
     def self.prepended(base)
       base.helper ::SolidusBraintree::BraintreeCheckoutHelper
     end
 
-    ::Spree::OrdersController.prepend(self) if SolidusSupport.frontend_available?
+    CartsController.prepend(self)
```

Comment:
As the user owns `CartsController`, shouldn't this code be injected by the generator? Probably in a neat way through the inclusion of a module or whatever.

Reply:
@waiting-for-dev Yes, I think it should be possible to include the decorators to the controllers. Will give it a try :+1: